js/app.js

```diff
@@ -110,11 +113,89 @@ seattleCenter.salesRender();
 alki.salesRender();
 capitolHill.salesRender();
 //I'm CALLING THE TOTAL
-firstandPike.totalSales();
-seaTacAirport.totalSales();
-seattleCenter.totalSales();
-alki.totalSales();
-capitolHill.totalSales();
+// firstandPike.totalSales();
+// seaTacAirport.totalSales();
+// seattleCenter.totalSales();
+// alki.totalSales();
+// capitolHill.totalSales();
+
+
+//EVENTS
+
+
+function firstTry(event){
+  event.preventDefault();
+  console.log('the form was submitted');
+
+  var firstElement = event.target;
+  var newInput = new CookieStore(firstElement.location.value, Number(firstElement.min.value),Number(firstElement.max.value),Number(firstElement.average.value));
+  console.log(newInput);
+  newInput.cookieSales();
+  newInput.storeSales();
+  newInput.salesRender();
+
+
+}
+
+var locationFormElement = document.getElementById('formElement');
+locationFormElement.addEventListener('submit', firstTry);
+
+
+function cookiesSoldPerStorePerHour() {
+  //I'm going hour per hour
+  //THIS IS THE SPIRAL PUNCH, THE HORIZONTAL LOOP, I'm making it walk.
+  for(var hour = 0; hour < storeHours.length; hour++){
+    //I'm going to...
+    //we're putting the var here because it has to be considered by both loops. If it was living in the inner loop, it would only be running in it. THAT IS WHY IT HAS TO BE OUTSIDE.
+    var sumOfCookies = 0;
+
+    //This is the vertical LOOP
+    for(var i = 0; i < allStores.length; i++){
+      //hour will give me the different hours not just the first column.
+      sumOfCookies += allStores[i].cookiesSoldPerHour[hour];
+
+    }
+    hourlyTotal.push(sumOfCookies);
+    console.log(sumOfCookies);
+  }
+
+}
+cookiesSoldPerStorePerHour();
+
+function renderFooter() {
+  var footRowElement= document.createElement('tr');
+  //I could have use datafooter en vez de total and then borrar var from loop.
```

Comment:
Spanglish in your comments is just fine, no need to change, but if you want to know English translations... en vez de translates to "instead of" and borrar translates to "erase" or "remove". (Or, if you don't want me to give feedback like this in the future, please let me know!)